src/EnergyPlus/StandardRatings.cc

```diff
@@ -6668,8 +6671,8 @@ namespace StandardRatings {
                                          RatedCOP,
                                          RegionNum,
                                          MinOATCompressor,
-                                         OATempCompressorOnOffBlank,
                                          OATempCompressorOn,
+                                         OATempCompressorOnOffBlank,
                                          DefrostControl);
```

Comment:
The function arguments needed to swap their order to match the argument's order in the function declaration.